tests/org/intellij/erlang/compilation/ErlangBuildTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2012-2015 Sergey Ignatov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.intellij.erlang.compilation;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.ModuleRootModificationUtil;
+import com.intellij.openapi.vfs.VirtualFile;
+
+import java.util.Collections;
+
+import static org.intellij.erlang.compilation.ErlangModuleTextGenerator.behaviour;
+import static org.intellij.erlang.compilation.ErlangModuleTextGenerator.module;
+import static org.intellij.erlang.compilation.ErlangModuleTextGenerator.pt;
+
+
+public class ErlangBuildTest extends ErlangCompilationTestBase {
+
+
+
+
+}
```

Comment:
Nope, you're wrong.  I'm really glad that you're perfect and do not miss anything.